yjit/src/codegen.rs

```diff
@@ -8293,6 +8293,9 @@ pub struct CodegenGlobals {
     /// Code for exiting back to the interpreter from the leave instruction
     leave_exit_code: CodePtr,
 
+    /// Code for compiling and jumping to the caller
+    leave_compile_code: CodePtr,
+
```

Comment:
Would it make sense to call this `leave_exception_code` instead of `leave_compile_code` so that its purpose is more clear?

Reply:
Sounds better :+1: https://github.com/ruby/ruby/pull/8171/commits/a83e4626a0564cd97a678ec0c3ef07c65620ce89